src/components/application_manager/rpc_plugins/app_service_rpc_plugin/CMakeLists.txt

```diff
@@ -28,7 +28,10 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-include_directories(include)
+include_directories(
+  include
+  ${COMPONENTS_DIR}/application_manager/rpc_plugins/sdl_rpc_plugin/include
```

Comment:
Not sure if theres a better way to handle this but it is used to handle subscribing the published service to OSCU notifications. I was unable to put this in the ASM since extracting the app extension requires an app id, which is not passed in to the `PublishAppService` function(besides the HMI does not have one)